lib/release_notes_bot/repositories.ex

```diff
@@ -0,0 +1,37 @@
+defmodule ReleaseNotesBot.Repositories do
+  @moduledoc """
+  Repo functions for Repoitories
+  """
+  alias ReleaseNotesBot.Repo
+  alias ReleaseNotesBot.Schema.Repository
+  alias __MODULE__
+
+  def create(params) do
+    %Repository{}
+    |> Repository.changeset(params)
+    |> Repo.insert()
+  end
+
+  def get_all do
+    Repo.all(Repository)
+  end
+
+  def get(param) do
+    Repo.get_by(Repository, param)
+  end
+
+  def existence(raw_repository) do
```

Comment:
This sounds like a `get_or_create` function. Not sure `existence` really makes sense here as it's sort of implying it's a predicate and not something that produces side effects.